### ISSUE TITLE
Slashes in branches generate unusable aliases with aliases.drushrc.php.stub

### ISSUE THREAD

**Issue by AlexSkrypnyk** [uselagoon/lagoon#214]
When a branch contains a slash (`/`) the alias generated by `aliases.drushrc.php.stub` would contain slash as well. This confuses drush and makes alias non-usable.

Example:
`feature/myfeature` would create `@feature/myfeature` drush alias.

Can this be changed to use `hyphen` instead: `@feature-myfeature`

Another question: can alias start with a project name: `@myproject-feature-myfeature`

**Comment by Schnitzel** [comment]
Thanks @alexdesignworks for this! Yes unfortunately the slashes are confusing for Drush, so converting them to slashes will be the right way.

Regarding Project: We are thinking about using the Drush SiteAlias Grouping Feature to prefix all Aliases with `lagoon.`. 
But that would not be the project name. What is your Idea behind the project name?

**Comment by AlexSkrypnyk** [comment]
@Schnitzel the project name would be how it appears in OpenShift UI, i.e., `[myproject] pr-1` would be `myproject-pr-1` as alias. Otherwise, `drush sa` shows per-branch aliases without project name among other non-lagoon drush aliases which makes it really confusing.

But this 'project in the name' feature is something for another ticket, I guess. Currently, we just need to fix slashes to unblock usage.

**Comment by rtprio** [comment]
Looks like I created a duplicate #484

**Comment by rtprio** [comment]
this was fixed in #484.
